docs/cluster-configuration.md

```diff
@@ -541,6 +541,14 @@ spec:
       managedDataSource: custom2
 ```
 
+## Workloads protection on uninstall
+
+`UninstallStrategy` defines how to proceed on uninstall when workloads (VirtualMachines, DataVolumes) still exist:
+- `BlockUninstallIfWorkloadsExist` will prevent the CR from being removed when workloads still exist.
+- `RemoveWorkloads` will cause all the workloads to be cascading deleted on uninstall.
```

Comment:
Please add some more text, to explain the meaning of deletion the HyuperConverged CR when this option is in use.

Reply:
done